tests/conftest.py

```diff
@@ -32,15 +33,22 @@ def setup_hotfix_check(request, ansible_module):
     """This fixture is used for installing hofix package and modifying foreman file.
     This fixture is used in test_positive_check_hotfix_installed_with_hotfix of test_health.py
     """
-    file = ansible_module.find(
-        paths="/opt/theforeman/tfm/root/usr/share/gems/gems/",
-        patterns="fog-vsphere-*",
-        file_type="directory",
-    )
-    dpath = file.values()[0]["files"][0]["path"]
-    fpath = dpath + "/lib/fog/vsphere/requests/compute/list_clusters.rb"
+    if rhel7():
+        file = ansible_module.find(
+            paths="/opt/theforeman/tfm/root/usr/share/gems/gems/",
+            patterns="fog-vsphere-*",
+            file_type="directory",
+        )
+        dpath = file.values()[0]["files"][0]["path"]
+        # RHEL7 path for list_clusters.rb
+        fpath = dpath + "/lib/fog/vsphere/requests/compute/list_clusters.rb"
+    else:
+        # RHEL8 path for list_clusters.rb
+        fpath = (
+            "/usr/share/gems/gems/fog-vsphere-3.5.0/"
+            "lib/fog/vsphere/requests/compute/list_clusters.rb"
+        )
```

Comment:
Here change seems to be only for ruby gems location like,
for `el7` - `/opt/theforeman/tfm/root/usr/share/gems/gems/`
for `el8` - `/usr/share/gems/gems/`

So I think instead of putting these commands in if-else, why not have constant for this path change in testfm constants for el7 vs el8, so it can be used by other tests(maybe in future) as well, what do you think?

Reply:
yeah, that would be great, I'll update it